examples/react/playground/src/stories/AuthenticationSubmitted.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { AuthenticationSubmitted } from '@w3ui/react'
+import '@w3ui/react/src/styles/authenticator.css'
+
+export default {
+  title: 'w3ui/AuthenticationSubmitted',
+  component: AuthenticationSubmitted,
+  tags: ['autodocs'],
+  argTypes: {
+  },
+  parameters: {
+    backgrounds: {
+      default: 'dark'
```

Comment:
this is neat!

Reply:
ooh let's set it in the defaults rather than per story for now
https://storybook.js.org/docs/react/essentials/backgrounds#configuration

Reply:
while we're noodling with defaults, _no one wants to see times new roman_ lets set at least a nice default font via
https://storybook.js.org/docs/react/configure/story-rendering#adding-to-body

Reply:
so real. will do!